bundles/org.palladiosimulator.dataflow.nodecharacteristics/META-INF/MANIFEST.MF

```diff
@@ -7,7 +7,7 @@ Bundle-Version: 0.1.0.qualifier
 Bundle-ClassPath: .
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
-Bundle-RequiredExecutionEnvironment: JavaSE-11
+Bundle-RequiredExecutionEnvironment: J2SE-1.5
```

Comment:
Change to `JavaSE-17`

Reply:
Should be resolved by the most recent commit